examples/airline/configs/agents.py

```diff
@@ -0,0 +1,93 @@
+from configs.tools import *
+from data.routines.baggage.policies import *
+from data.routines.flight_modification.policies import *
+from data.routines.prompts import STARTER_PROMPT
+
+from openai_runner import Agent
+
+
+def transfer_to_flight_modification():
+    """Call this function when a user needs to be transferred to flight modification agent."""
+    return flight_modification
+
+
+def transfer_to_flight_cancel():
+    return flight_cancel
+
+
+def transfer_to_flight_change():
+    return flight_change
+
+
+def transfer_to_lost_baggage():
+    return lost_baggage
+
+
+def transfer_to_triage():
+    """Call this function when a user needs to be transferred to a different agent and a different policy.
+    For instance, if a user is asking about a topic that is not handled by the current agent, call this function.
+    """
+    return triage_agent
+
+
+def triage_instructions(context_variables):
+    customer_context = context_variables.get("customer_context", None)
```

Comment:
**Suggestion:** Add error handling in `triage_instructions` to manage cases where `context_variables` are missing expected keys, enhancing robustness and preventing runtime errors. [general, importance: 7]
```suggestion
customer_context = context_variables.get("customer_context", "No customer context available.")
flight_context = context_variables.get("flight_context", "No flight context available.")
```